sdk/src/main/java/com/mapbox/maps/MapView.kt

```diff
@@ -115,7 +116,8 @@ open class MapView : FrameLayout, MapPluginProviderDelegate, MapControllable {
         MapAttributeParser.parseMapOptions(typedArray, context.resources.displayMetrics.density)
       val cameraOptions = CameraAttributeParser.parseCameraOptions(typedArray)
       val textureView = typedArray.getInt(R.styleable.mapbox_MapView_mapbox_mapSurface, 0) != 0
-      return MapInitOptions(context, resourceOptions, mapOptions, attrs = attrs).also {
+      val styleUri = typedArray.getString(R.styleable.mapbox_MapView_mapbox_styleUri)
```

Comment:
if the mapbox_MapView_mapbox_styleUri doesn't exit, then the styleUri will be set to null, I think we need to use a default value MAPBOX_STREETS  here, and figure out a way to set an empty StyleUri.

Reply:
A potential solution would be checking the existence of mapbox_MapView_mapbox_styleUri, if user has specifically set styleUri="" in the xml, we set it to null, if user doesn't specify styleUri in xml, we use default value(MAPBOX_STREETS).

And please also add the usage of this attribute one of our examples to test it out.

Reply:
Good catch and solution here 👍